packages/service/src/utils/user-documents.ts

```diff
@@ -0,0 +1,9 @@
+import { db, pool } from "../db";
+
+export const createUserDocument = (docId: string, userId: string) =>
+  db
+    .insert("user_document", {
+      document_id: docId,
+      user_id: userId,
+    })
+    .run(pool);
```

Comment:
Should this even be a separate function? Maybe this could move to the createDocument function in documents.ts?